### ISSUE TITLE
Latest commit does not compute... ehm, compile

### ISSUE THREAD

**Issue by MrPsyMan** [reicast/reicast-emulator#375]
It's been a bit messy for a while now. I remember having to edit build.xml cause it had some ndk environmental variables that messed with packaging at some point (when building the r6 RC).

Well, attempting to build the most recent commit shows links wrong paths and the likes.

Apparently someone else wants to troll the rest of us. Trolling the rest of you with unreasonable actions is supposed to be my job so please try to refrain from doing it.

TLDR version: Could the one who broke it take the time to re-fix it? Thanks.
